### ISSUE TITLE
Did not work with me

### ISSUE THREAD

**Issue by tamimology** [nikarh/jellyfin-plugin-authelia#43]
I have done as the steps in your repo say, and still, I cannot see Authelia login yet

Below is the plugin config from Jellyfin
![image](https://github.com/nikarh/jellyfin-plugin-authelia/assets/90429593/eeac92e4-13bb-41f0-a934-e8ee63d22abc)

This is the usual Jellyfin login page when I enter my url, once I login, I am in with no 2FA via Authelia

![image](https://github.com/nikarh/jellyfin-plugin-authelia/assets/90429593/f4cf5ccf-b138-4dcc-a1d8-bd6b648b56dd)

What am I missing here?

Note, I have Authelia working just perfect with other services, but not this

**Comment by nikarh** [comment]
Hi there!

This is mentioned in the [description](https://github.com/nikarh/jellyfin-plugin-authelia#description) of this plugin.

There are two ways you can technically integrate Jellyfin with Authelia.
1. The first approach would be redirected to the Authelia login page and back to Jellyfin. If you want that, you should use [SSO plugin](https://github.com/9p4/jellyfin-plugin-sso) instead of this. This way you would get working 2FA, since all that is happening on Authelia's side. This approach does not work for me, since I am not only using the web UI for Jellyfin but also obscure Android clients, which would know nothing about the redirecting thing and will not work with the SSO plugin, since it is not part of Jellyfin API authentication flow.
2. The alternative approach is using Jellyfin UI and API for login/password, and using Jellyfin as the authentication provider in the backend. This is the approach taken by this plugin. It fits the natural auth flow and works with alternative Jellyfin clients, but since Jellyfin auth flow has no API flow for 2FA, it will log you in without 2FA.

tldr; If you need 2FA, you're better off with [SSO plugin](https://github.com/9p4/jellyfin-plugin-sso)

**Comment by nikarh** [comment]
I guess I can add the note about 2FA to the plugin configuration page to not cause confusion
